fastly/products/brotlicompression/api_test.go

```diff
@@ -17,7 +17,6 @@ var functionalTests = []*test_utils.FunctionalTest{
 		Phase:         "ensure disabled before testing",
 		OpFn:          brotlicompression.Disable,
 		ServiceID:     serviceID,
-		IgnoreFailure: true,
```

Comment:
As long the request has the proper access rights that endpoint always returns a `204` no matter what.